tox.ini

```diff
@@ -85,19 +85,19 @@ commands =
     -n -W --keep-going \
     -d "{temp_dir}/.doctrees" \
     . \
-    "{envdir}/docs_out"
-
-  # Print out the output docs dir and a way to serve html:
-  -{envpython} -c\
-  'import pathlib;\
-  docs_dir = pathlib.Path(r"{envdir}") / "docs_out";\
-  index_file = docs_dir / "index.html";\
-  print("\n" + "=" * 120 +\
-  f"\n\nDocumentation available under:\n\n\
-  \tfile://\{index_file\}\n\nTo serve docs, use\n\n\
-  \t$ python3 -m http.server --directory \
-  \N\{QUOTATION MARK\}\{docs_dir\}\N\{QUOTATION MARK\} 0\n\n" +\
-  "=" * 120)'
+    {posargs:{envdir}/docs_out}
```

Comment:
I see you didn't migrate the `-t is_unversioned` bit. This is fine for now as it's out of the scope, but would be good to do in a follow-up. It's particularly useful when `setuptools-scm` is used as the version string being dynamic prevents Sphinx from reusing cache and we can optimize it by setting it to something static in local dev envs.